apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Tests/Logging/GoogleLoggerTest.cs

```diff
@@ -441,23 +443,20 @@ public void Log_NoFormatParams()
         public void Log_Trace()
         {
             string traceId = "external_trace_id";
-            string spanId = "external_span_id";
+            ulong spanId = 1;
             string fullTraceName = TraceTarget.ForProject(ProjectId).GetFullTraceName(traceId);
 
             Predicate<IEnumerable<LogEntry>> matcher = logEntries =>
             {
                 LogEntry entry = logEntries.Single();
                 return entry.LogName == new LogName(ProjectId, BaseLogName).ToString() &&
                     entry.Trace == fullTraceName &&
-                    entry.SpanId == spanId;
+                    entry.SpanId == $"{spanId:x16}";
```

Comment:
It's unclear to me why this had to change. Is it because we're not requiring the external span IDs to be 64-bit integers?

Reply:
Yes, the now obsolete TraceContextForLogEntry has a string for SpanId, and docs saying that it should be in base 16. But now we use the new ITraceContext where span ID is a ulong, and for setting it in the entry we are the ones who have to convert it to base 16.